package.json

```diff
@@ -28,17 +28,19 @@
     "csrf": "^3.0.2",
     "debug": "^2.1.2",
     "express": "^4.12.2",
+    "hof-form-controller": "^3.0.1",
     "hof-model": "^2.0.0",
-    "hof-controllers": "^6.0.0",
     "hogan.js": "^3.0.2",
     "i18n-lookup": "^0.1.0",
-    "lodash": "^4.17.2"
+    "lodash": "^4.17.2",
+    "mixwith": "^0.1.1"
   },
   "devDependencies": {
     "chai": "^3.5.0",
     "eslint-config-homeoffice": "^2.0.0",
     "istanbul": "^0.4.3",
     "mocha": "^3.1.2",
+    "proxyquire": "^1.7.11",
```

Comment:
This doesn't seem to be used anywhere - or at least hasn't been added elsewhere in this PR - is it a linting thing?